ansible/roles/ecryptfs/tasks/main.yml

```diff
@@ -4,8 +4,8 @@
   apt: name=ecryptfs-utils state=present
 
 - name: Check for already mounted drive
-  shell: cat /etc/fstab
-  register: fstab_contents
+  shell: cat /etc/mtab
+  register: mtab_contents
```

Comment:
fstab stores filesystems available to mount, mtab stores the ones actually mounted.  The second time this is run after a restart the line will be in fstab, and skip the mounting process. http://www.linuxquestions.org/questions/linux-newbie-8/difference-between-mtab-and-fstab-924190/